BucStop/wwwroot/js/site.js

```diff
@@ -2,3 +2,42 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+var game = document.getElementById("game");
+if (game != null) { //used so the browser doesn't freak out if not at a game page
+
+    const canvasSwipe = document.getElementById('game');
```

Comment:
Could the game variable on line 5 not be used instead of this? Since this is basically the same thing as ling 5 making it a constant instead.